tests/unit/src/test/scala/tests/markdown/IndentedMarkdownSuite.scala

```diff
@@ -0,0 +1,86 @@
+package tests.markdown
+
+class IndentedMarkdownSuite extends BaseMarkdownSuite {
+
+  check(
+    "2 whitespace",
+    """
+      |  ```scala
+      |  val msg = "Hello!"
+      |  println(msg)
+      |  ```
+    """.stripMargin,
+    """
+      |  ```scala
+      |  val msg = "Hello!"
+      |  println(msg)
```

Comment:
Shouldn't expected have the msg printed? The same in the other tests.

Reply:
what do you mean?
`check` it a wrapper around `checkCompile` as far as i can see. so nothing is executed/printed